packages/nc-gui/components/account/Token.vue

```diff
@@ -158,22 +165,37 @@ const descriptionInput: VNodeRef = (el) => (el as HTMLInputElement)?.focus()
             v-if="showNewTokenModal"
             class="flex gap-5 px-3 py-3.5 text-gray-500 font-medium text-3.5 w-full nc-token-generate"
           >
-            <a-input
-              :ref="descriptionInput"
-              v-model:value="selectedTokenData.description"
-              type="text"
-              class="!rounded-lg !py-2"
-              placeholder="Token Name"
-              data-testid="nc-token-input"
-            />
+            <div class="flex flex-col w-full">
+              <a-input
+                :ref="descriptionInput"
+                v-model:value="selectedTokenData.description"
+                type="text"
+                class="!rounded-lg !py-2"
+                placeholder="Token Name"
+                data-testid="nc-token-input"
+              />
+              <span v-if="isTextEmpty" class="text-red-500 text-xs font-light mt-1.5 ml-1">token name should not be empty</span>
```

Comment:
I saw some call `token name` and some call `token desc`. Are they interchangeable? If so, better keep it consistent with the naming to avoid confusion.